webapp/javascript/components/Heatmap/index.tsx

```diff
@@ -25,15 +25,15 @@ interface HeatmapProps {
     startT: number,
     endT: number
   ) => void;
-  timezone: string;
   sampleRate: number;
+  timezone: string;
```

Comment:
where's this used?

Reply:
https://github.com/pyroscope-io/pyroscope/blob/main/webapp/javascript/components/Heatmap/utils.ts#L127

Reply:
i just reordered props in asc order